example/samples/install_photon_os_payload_full.json

```diff
@@ -0,0 +1,83 @@
+{
+    "options": {
+        "install-os": {
+            "version": "1.0",
+            "repo": "http://172.31.128.1:9080/photon/1.0"
+            "installType": "full",
```

Comment:
I have no idea about it. I tried to delete it and re-enter in English keyboard several times, but git diff shows nothing.